docs/source/markdown/podman-create.1.md

```diff
@@ -22,6 +22,49 @@ Default settings for flags are defined in `containers.conf`. Most settings for
 remote connections use the server's containers.conf, except when documented in
 man pages.
 
+## IMAGE
+
+  The image is specified using transport:path format. If no transport is specified, the `docker` (container registry)
+transport will be used by default. For remote Podman, `docker` is the only allowed transport."
+
+  **dir:**_path_
+  An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files. This
+is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
+
+    $ podman save --format docker-dir fedora -o /tmp/fedora
+    $ podman create dir:/tmp/fedora echo hello
+
+  **docker://**_docker-reference_ (Default)
+  An image reference stored in  a remote container image registry. Example: "quay.io/podman/stable:latest".
+The reference can include a path to a specific registry; if it does not, the registries listed in
+registries.conf will be queried to find a matching image. By default, credentials from podman login (stored at
+$XDG_RUNTIME_DIR/containers/auth.json by default) will  be used to authenticate; if these cannot be found, we
+will fall back to using credentials in $HOME/.docker/config.json.
+
+    $ podman create registry.fedoraproject.org/fedora:latest echo hello
+
+  **docker-archive:**_path_[**:**_docker-reference_]
+An image stored in the `docker save` formatted file.  _docker-reference_ is only used when creating such a
+file, and it must not contain a digest.
+
+    $ podman save --format docker-archive fedora -o /tmp/fedora
+    $ podman create docker-archive:/tmp/fedora echo hello
+
+  **docker-daemon:**_docker-reference_
+  An image in _docker-reference_ format stored in the docker daemon internal storage.  _docker-reference_ must
+contain either a tag.  Alternatively, when reading images, the format can also be docker-daemon:algo:digest (an
```

Comment:
I'm not sure what to do with "either a tag"?  At the very least an "or" is missing or perhaps it's a really bad cut/paste?